index.js

```diff
@@ -14,14 +20,14 @@ module.exports = {
       debugMode = process.env.EMBER_ENV === 'development';
     }
 
-    if (!process.env.EMBER_CLI_FASTBOOT) {
-      if (debugMode) {
-        app.import('bower_components/plupload/js/moxie.js');
-        app.import('bower_components/plupload/js/plupload.dev.js');
-      } else {
-        app.import('bower_components/plupload/js/plupload.full.min.js');
-      }
+
+    if (debugMode) {
+      app.import('vendor/plupload/moxie.js');
```

Comment:
@tim-evans happy to change, but curious to understand why you want to chain for `Moxie.swf` too since that can be imported via `bower`.